### PR TITLE
Added option to control who sees player (everybody or only logged in users)

### PR DESCRIPTION
**Pull request by gingerbeardman**

### CODE REVIEW

**Comment by hlashbrooke** [comment]
I'm happy with the implementation of this in terms of how you've written the code (it's simple enough to integrate without affecting anything else in the plugin) - I'm just hesitant to bring a feature like this into core as I feel like it's something that only a few people would use. I may be thoroughly incorrect in that assumption though, so I need to do a bit of research to determine whether that's fair or not.

File: includes/class-ssp-frontend.php [hunk 1]
```diff
@@ -171,7 +171,14 @@ public function content_meta_data( $content = '' ) {
 
 		$podcast_post_types = ssp_post_types( true );
 
-		if ( in_array( $post->post_type, $podcast_post_types ) && ! is_feed() && ! isset( $_GET['feed'] ) ) {
+		$player_visibility = get_option( 'ss_podcasting_player_content_visibility' );
```
Comment (hlashbrooke) [inline includes/class-ssp-frontend.php:174]: This `get_option` call needs a default value of `all` just to make sure that it catches every scenario ([function docs here](https://developer.wordpress.org/reference/functions/get_option/)). Other than that, this code is all good with me.

Comment (gingerbeardman) [reply includes/class-ssp-frontend.php:174]: good spot

**Comment by gingerbeardman** [comment]
Obviously your call.

My use case is to put a podcast behind a login wall for members of a site. The podcast is not listed on iTunes and so is used as one of the big reasons to join the site.

This seemed like the cleanest way of doing it. If you do not accept it I'd appreciate some help with how I might go about achieving the same thing using only filters in function.php something I've not done before.

**Comment by hlashbrooke** [comment]
Let me give this some thought and I'll see if it's something I want to include in the next release. Thanks!

**Comment by hlashbrooke** [comment]
After giving this some thought, I think I'll go ahead and merge this to include it in the next release (v1.15) with a few other additional features that I have wrapped up.

Thanks for the contribution!

**Pull request merged**